Formula/coreutils.rb

```diff
@@ -50,6 +53,8 @@ def breaks_macos_users
     %w[dir dircolors vdir]
   end
 
+  patch :DATA
```

Comment:
Add a comment explaining what the patch is for and when it can be removed, please.

Reply:
the comment is in the patch itself...

Reply:
6aa86dd7b8b b865d978858